Example/Tests/StringScalar/TrimmedPrefixStringTests.swift

```diff
@@ -17,29 +17,34 @@ final class TrimmedPrefixStringTests: XCTestCase {
     func testTrimsExistentPrefix() {
         Array<
             (
-                String,
-                String
+                String, //Full string
```

Comment:
Please replace comments with tuple names

Reply:
Well it is pretty straightforward even without comment